### PR TITLE
Update circleci node version

### PR DESCRIPTION
**Pull request by elenatorro**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ orbs:
 jobs:
   test-e2e:
     docker:
-      - image: circleci/node:10-browsers
+      - image: circleci/node:10.15.1-browsers
```
Comment (VictorVelarde) [inline .circleci/config.yml]: It looks like we need a little image name: 10.15.1-jessie or 10.15.1-stretch (https://github.com/CircleCI-Public/circleci-dockerfiles/tree/master/node/images) and then '-browsers'

Comment (elenatorro) [reply .circleci/config.yml]: I see!! 👍 let's try this

**Comment by elenatorro** [comment]
Fixed this by using a different docker image. Adding @ethervoid to double check this is correct 👍

**Comment by ethervoid** [comment]
@elenatorro My changes were merged to `develop` branch and they differ a bit. Now we have [two test-e2e tasks](https://github.com/CartoDB/carto-vl/blob/develop/.circleci/config.yml#L11) one for pg10 and another for pg11

I suggest you to merge develop in master or point your changes to `develop`

**Comment by VictorVelarde** [comment]
👍

**Comment by ethervoid** [comment]
:shipit:

**Pull request merged**